app/views/devise/passwords/edit.html.erb

```diff
@@ -2,7 +2,7 @@
   Change your password - <%= t('page_title') %>
 <% end %>
 
-<h1 class="govuk-heading-l">Change your password</h1>
+<h1 class="govuk-heading-l">Create your password </h1>
```

Comment:
Really tiny point but I wonder if this should say either "change" or "create" depending on whether this is new or a reset? "Create" does sort of work for a reset though so maybe it doesn't matter.

Reply:
I orignally had an `if` statement in the .erb to do exactly that, but then I decided the added complexity wasn't really worth it. Maybe I will reinstate, as you've also thought this :)